console/src/app/modules/accounts-card/accounts-card.component.ts

```diff
@@ -45,21 +45,27 @@ export class AccountsCardComponent implements OnInit {
         }
     }
 
-    public selectAccount(loginHint?: string, idToken?: string): void {
+    public selectAccount(loginName?: string): void {
         const configWithPrompt: Partial<AuthConfig> = {
             customQueryParams: {
-                prompt: 'select_account',
+                // prompt: 'select_account',
```

Comment:
do we want to remove the prompt param?

Reply:
yes, oidc standard states you can always send a login hint